stable/drone/README.md

```diff
@@ -60,37 +63,15 @@ The following table lists the configurable parameters of the drone charts and th
-| `agent.readinessProbe.initialDelaySeconds` | Delay before readiness probe is initiated                                     | 0                            |
-| `agent.readinessProbe.periodSeconds` | How often to perform the probe                                                      | 10                           |
-| `agent.readinessProbe.timeoutSeconds` | When the probe times out                                                           | 1                            |
-| `agent.readinessProbe.successThreshold` | Minimum consecutive successes for the probe to be considered successful after having failed. | 1                |
-| `agent.readinessProbe.failureThreshold` | Minimum consecutive failures for the probe to be considered failed after having succeeded. | 3                  |
-| `dind.enabled`              | Enable or disable **DinD**                                                                    | `true`                      |
-| `dind.driver`               | **DinD** storage driver                                                                       | `overlay2`                  |
-| `dind.resources`            | **DinD** pod resource requests & limits                                                       | `{}`                        |
-| `dind.env`                  | **DinD** environment variables                                                                | `nil`                       |
-| `dind.command`              | **DinD** custom command instead of default entry point                                        | `nil`                       |
-| `dind.args`                 | **DinD** arguments for custom command or entry point                                          | `nil`                       |
-| `metrics.prometheus.enabled` | Enable Prometheus metrics endpoint                                                          | `false`                     |
 | `persistence.enabled`       | Use a PVC to persist data                                                                     | `true`                      |
 | `persistence.existingClaim` | Use an existing PVC to persist data                                                           | `nil`                       |
 | `persistence.storageClass`  | Storage class of backing PVC                                                                  | `nil`                       |
 | `persistence.accessMode`    | Use volume as ReadOnly or ReadWrite                                                           | `ReadWriteOnce`             |
 | `persistence.size`          | Size of data volume                                                                           | `1Gi`                       |
-| `sharedSecret`              | Drone server and agent shared secret (Note: The Default random value changes on every `helm upgrade` causing a rolling update of server and agents) | `(random value)`            |
```

Comment:
You have removed this from the values file but it is still referenced in the secrets.yaml and used for DRONE_RPC_SECRET.